manifests/params.pp

```diff
@@ -0,0 +1,15 @@
+class etherpad::params {
+  $default_padoptions = {
+    'nocolors'         => 'false',
```

Comment:
puppet-lint doesn't like these quoted booleans. I wonder if we should change the datatypes to `Boolean` for these but I don't know if these really are booleans in etherpad.